### PR TITLE
Add chore and feature templates

### PR DESCRIPTION
**Pull request by yaelharel**
Signed-off-by: Yael Harel <yharel@vmware.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/chore.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+labels: type/chore, status/triage
+assignees: ''
+
+---
+
+### Description
+<!-- A concise description of why this chore matters, who will enjoy it and how. -->
+
+### Proposed solution
+<!-- A clear and concise description of how do you think the chore should be implemented. -->
```
Comment (natalieparellano) [inline .github/ISSUE_TEMPLATE/chore.md:14]:
```suggestion
<!-- A clear and concise description of how you think the chore should be implemented. -->
```

File: .github/ISSUE_TEMPLATE/chore.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Chore
+about: Suggest a chore that will help contributors and doesn't affect end users.
```
Comment (natalieparellano) [inline .github/ISSUE_TEMPLATE/chore.md:3]:
```suggestion
about: Suggest a chore that will help contributors and doesn't affect end users
```

Comment (natalieparellano) [reply .github/ISSUE_TEMPLATE/chore.md:3]: (I don't know... the other one didn't have a period)

**Comment by natalieparellano** [comment]
This is great! I left a couple nits.

**Review by natalieparellano — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/chore.md (hunk 1 above)
Comment (yaelharel) [reply .github/ISSUE_TEMPLATE/chore.md]: Please see https://github.com/buildpacks/lifecycle/pull/581/commits/a15705e2af10bfc1bb15cbc4a29d592b8e4cc063.

File: .github/ISSUE_TEMPLATE/chore.md (hunk 2 above)
Comment (yaelharel) [reply .github/ISSUE_TEMPLATE/chore.md]: Please see https://github.com/buildpacks/lifecycle/pull/581/commits/a15705e2af10bfc1bb15cbc4a29d592b8e4cc063.

**Pull request merged**